backend/groq/pdf_to_text.py

```diff
@@ -12,11 +12,10 @@
 import textwrap
 from typing import List
 
-# Update the path to include backend directory
-ROOT = os.path.abspath(os.path.dirname(__file__))
+ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), '..'))
```

Comment:
Two levels of depth for 2 sub-folders
ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..'))